src/snapshots/records/error_malformed_syntax.md

```diff
@@ -0,0 +1,52 @@
+# META
+~~~ini
+description=Malformed record syntax (error case)
+type=expr
+~~~
+# SOURCE
+~~~roc
+{ name: "Alice", : 30, , email: , active Bool.true, "invalid": value, 42: "number key", : }
+~~~
+~~~
```

Comment:
Sometimes it generates two lines of this section end... I haven't been able to figure out why yet. (unrelated to this PR)